src/mock-data/author.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
minor. I still prefer to the synthetic data over real trace for mocking. The content of the response does not matter. Only the response data type does.

Reply:
I agree to use synthetic data for mocking with essential part for UI testing.

Updated.